mdp/nodes/sfa_nodes.py

```diff
@@ -369,6 +376,170 @@ def get_eta_values(self, t=1):
         return self._refcast(t / (2 * numx.pi) * numx.sqrt(self.d))
 
 
+class VartimeSFANode(SFANode):
+    """
+    Extract the slowly varying components from the input data.
+    This node can be understood as a generalization to the *SFANode*.
+
+    In particular this nodes numerically computes the integrals involved in
```

Comment:
comma after "particular" and "nodes" shouldn't be plural I guess. (otherwise "computes" is wrong)